app.js

```diff
@@ -1,11 +1,83 @@
 /* Imports */
-
+const climateSelect = document.getElementById('climate-select');
+const architectureSelect = document.getElementById('architecture-select');
+const attractionsSelect = document.getElementById('attractions-select');
+const climateImage = document.getElementById('climate-image');
+const architectureImage = document.getElementById('architecture-image');
+const attractionsImage = document.getElementById('attractions-image');
+const sloganInput = document.getElementById('slogan-input');
+const addButton = document.getElementById('add-button');
+const sloganList = document.getElementById('slogan-list');
+const nameInput = document.getElementById('name-input');
+const nameList = document.getElementById('name-list');
 /* Get DOM Elements */
 
 /* State */
+const destination = {
+    names: [],
+    climate: 'tropical',
+    architecture: 'city',
+    attractions: 'shrine',
+    slogans: [],
+};
 
 /* Events */
+nameInput.addEventListener('input', () => {
+    destination.name = nameInput.value;
+
+    displayDestination();
```

Comment:
so you're calling ```displayDestination()``` here, which is good, you want to take the input value from the user interaction, store it in the variable, and rerender. however when we get down below on line 49...